### DISCUSSION TITLE
topic filter?

### DISCUSSION THREAD

**Discussion by jpetrakis** [Monika-After-Story/MonikaModDev#9029] [issue discussions]
first post here and just want to say I love the mod and it's very fun so far; I'm excited to explore and discover new things.

one thing that bothers me is that it seems that Monika is very health-conscience in the sense that she food shames. in the instance of "don't drink too much soda" and "have a salad" when I declare I'm hungry, it can be really triggering for someone with an ED and I wanted to know if there was any way to filter out these specific topics as to not be taxing on my mental health.

any guidance or advice would be greatly appreciated

**Comment by multimokia** [comment]
While admittedly not exactly the best system, next time she brings a topic like that up, hit the `X` key on your keyboard and she'll avoid mentioning that topic again in future, and the salad one comes from the `I feel...hungry` mood

That said the soda one is a one-time topic so she shouldn't end up bringing it up regardless.
